### DISCUSSION TITLE
How to consume fields named as restricted Java keywords with Quarkus Reactive Rest Client?

### DISCUSSION THREAD

**Discussion by jack-farley** [quarkusio/quarkus#30693] [Q&A] [answered]
I'm trying to use the Reactive Rest Client in Quarkus to fetch some data from an external API. However, in the JSON response of this API, one of the fields is named 'long' (i.e. longitude). I obviously can't create a Java object with a field named 'long' to use as the return type of an endpoint in my REST client, and I can't find an easy way around this. Any help would be much appreciated.

[a comment by an automated account is not shown]

**Comment by geoand** [comment, marked as answer]
If you use `quarkus-rest-client-reactive-jackson`, then you can use `@JsonProperty("long")` on the field of the POJO in question

**Reply by jack-farley** [reply]
Thanks!

**Reply by geoand** [reply]
🙏🏼
